ci/pipeline.yml

```diff
@@ -48,8 +49,8 @@ source:
 groups:
   - name: galoy
     jobs:
-      - test-integration
-      - test-e2e
+      - integration-tests
+      - e2e-tests
       - test-unit
```

Comment:
This should also be changed to maintain uniformity in naming?

Reply:
```
 - #@ test_on_docker_host("e2e-tests")
 ```
 
 the way the fragment works is it uses the argument (`e2e-tests`) both as the concourse job name as well the name of the container it will run.